cdap-docs/examples-manual/source/examples/hello-world.rst

```diff
@@ -53,116 +53,114 @@ by the length of the name. We will see below how to retrieve these metrics using
    :start-line: 11
 
+
 Running the Example
 ===================
 
 Starting the Flow
-------------------------------
+-----------------
 
 Once the application is deployed:
 
-- Click on the *Process* button in the left sidebar of the CDAP UI,
-  then click ``WhoFlow`` in the *Process* page to get to the
-  Flow detail page, then click the *Start* button; or
-- From the Standalone CDAP SDK directory, use the Command Line Interface:
+- Go to the *HelloWorld* `application overview page 
+  <http://localhost:9999/ns/default/apps/HelloWorld/overview/status>`__,
+  click ``WhoFlow`` to get to the flow detail page, then click the *Start* button; or
+- From the Standalone CDAP SDK directory, use the Command Line Interface::
 
-  .. list-table::
-    :widths: 20 80
-    :stub-columns: 1
-
-    * - On Linux:
-      - ``$ ./bin/cdap-cli.sh start flow HelloWorld.WhoFlow``
-    * - On Windows:
-      - ``> bin\cdap-cli.bat start flow HelloWorld.WhoFlow``    
+    $ cdap-cli.sh start flow HelloWorld.WhoFlow
+  
+    Successfully started Flow 'WhoFlow' of application 'HelloWorld' with stored runtime arguments '{}'
 
 Starting the Service
 ------------------------------
 
 Once the application is deployed:
 
-- Click on ``HelloWorld`` in the Overview page of the CDAP UI to get to the
-  Application detail page, click ``Greeting`` in the *Service* pane to get to the
-  Service detail page, then click the *Start* button; or
-- From the Standalone CDAP SDK directory, use the Command Line Interface:
-
-  .. list-table::
-    :widths: 20 80
-    :stub-columns: 1
+- Go to the *HelloWorld* `application overview page 
+  <http://localhost:9999/ns/default/apps/HelloWorld/overview/status>`__,
+  click ``Greeting`` to get to the service detail page, then click the *Start* button; or
+- From the Standalone CDAP SDK directory, use the Command Line Interface::
 
-    * - On Linux:
-      - ``$ ./bin/cdap-cli.sh start service HelloWorld.Greeting``
-    * - On Windows:
-      - ``> bin\cdap-cli.bat start service HelloWorld.Greeting``    
+    $ cdap-cli.sh start service HelloWorld.Greeting
+    
+    Successfully started Service 'Greeting' of application 'HelloWorld' with stored runtime arguments '{}'
 
 Injecting a Name
-------------------------------
+----------------
+
+.. |who-flow| replace:: *WhoFlow*
+.. _who-flow: http://localhost:9999/ns/default/apps/HelloWorld/programs/flows/WhoFlow/runs
 
-In the Application's detail page, under *Process*, click on *WhoFlow*. This takes you to
-the flow details page. (If you haven't already started the Flow, click on the *Start*
-button in the right-side, below the green arrow.) The Flow's label will read *Running*
+In the Application's detail page, click on |who-flow|_. This takes you to
+the flow details page. (If you haven't already started the flow, click on the *Start*
+button in the right-side, below the green arrow.) The flow's *status* will read *Running*
```

Comment:
Should you specify `RUNNING`, because it's actually all capitalized in the UI and also curl.